motionplan/collision_test.go

```diff
@@ -121,3 +121,17 @@ func TestUniqueCollisions(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, collisionListsAlmostEqual(cs.Collisions(), expectedCollisions[:1]), test.ShouldBeTrue)
 }
+
+func TestXArm7(t *testing.T) {
```

Comment:
It seems unpleasant that we have a specific arm component test that isn't in the `component` part of the codebase. I assume you were stuck with the import dependency issues on needing to bring in `motionplan` for this, right? I remember having to do something very similar with kinematic models derived from URDF sources and checking their correctness.

Not worth fixing here, but would it be on the table to split things into `yadda_test` packages? I believe I've seen that pattern elsewhere within engineering but I don't know if that's blessed or not. Before I do that legwork, do any of you have an opinion on this?

Reply:
>It seems unpleasant that we have a specific arm component test that isn't in the component part of the codebase. I assume you were stuck with the import dependency issues on needing to bring in motionplan for this.

Putting the test in the `xarm` package works for me.
My initial thought was to put this test in the `motion` plan package because we are testing for collisions.

Reply:
I also this think specific test needs to be updated to be more rigorous, ie testing with various poses (impossible poses and verified ones).

In that vein, a test for the ur5e also needs to be added.

Reply:
This is not how we want to check the exact values/locations of arm geometries at a couple different poses. This test is now removed. 

Pose check unit tests will be in `arm_test.go`